data/getting-started/nvidia.md

```diff
@@ -0,0 +1,8 @@
+# NVIDIA GPU Add-on
+
+Red Hat OpenShift Container Platform is a security-centric and enterprise-grade hardened Kubernetes platform for deploying and managing Kubernetes clusters at scale, developed and supported by Red Hat. Red Hat OpenShift Container Platform includes enhancements to Kubernetes so users can easily configure and use GPU resources for accelerating workloads such as deep learning.
+
+The NVIDIA GPU Operator uses the operator framework within Kubernetes to automate the management of all NVIDIA software components needed to provision GPU. These components include the NVIDIA drivers (to enable CUDA), Kubernetes device plugin for GPUs, the NVIDIA Container Toolkit, automatic node labelling using GFD, DCGM based monitoring and others.
```

Comment:
>automatic node labelling using GFD
Shouldn't this be NFD? As in `Node Feature Discovery`?

Reply:
I used this text from the documentation of the GPU operator itself. https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/introduction.html

Reply:
Mh, yeah it does seem to be different. Unsure about the correct usage in this case, since in our clusters what gets installed is the `NFD` operator along with the GPU add-on.
Perhaps @LaVLaS might know something more here